score/README.md

```diff
@@ -0,0 +1,36 @@
+# Torchbench Score
+
+Torchbench provides a normalized benchmark score similar to 'spec' or other computing benchmarks.
+
+This is a prototype.  Current status and limitations are described below.
+
+## Score versioning
+The score is versioned, meaning only a certain set of benchmarks are captured in a particular 
+version of the score (even if additional benchmarks are added to the suite).  The relative weight
+of each benchmark in the overall score is frozen, along with a normalization factor measured on
+a particular 'gold' machine with a particular PyTorch release.  The intent is to measure the effect
+of new pytorch versions on the same workloads using the same reference machine using a consistent
+benchmark configuration.
+
+## Computing the score
+To compute the current score, provide a score config and benchmark data produced by pytest with `--benchmark-json` or related arguments.
+`python compute_score.py --configuration <cfg> --benchmark_data_file <data-file> `
+Or, use `--benchmark_data_dir` instead, pointing to a directory containing multiple json files to compute a table of scores.
+
+## New score versions
+Periodically, as more workloads have been added to the torchbenchmark suite, or as changes to
+relative weights or categories have been proposed, a new score configuration should be generated
+rather than modifying an existing score definition. 
+
+See `python generate_score_config.py -h` 
+
+## Issues and Next Steps
+For accurate score comparisons, measurements should be computed on the same machine 
```

Comment:
This sounds fine for now. But in our past experience, this requirement is hard to maintain in large scale across a long period of time. As we get more benchmarks, we will likely use more machines of the same type, and we might have to migrate to a different pool from time to time. 

So a good approach is to offset the scores to stay the same during transition. We don't have to do it now, but it might be worth mentioning in the future directions.

Reply:
yea, i agree that we can't really limit to a single machine.  I have been thinking about defining a machine spec (cpu, gpu type, memory cap, potentially kernel version, etc.) and making sure that part matches even though the hostname is different.

Reply:
oh, and my original intent was that a score configuration was bound to a machine type, but every quarter we might take the latest pytorch release, more benchmark models, and possibly a different machine type, and use that to compute a new normalization file with the target set to match what the latest target was on the old score.

it would also be possible to support multiple machine types at the same time, just by maintaining different sets of norms for each machine type.